README.md

```diff
@@ -34,14 +34,17 @@ fn rdm_encipher(message: &str) -> (u16, String)
 You can try the lib using the `caesar-cli` tool:
 
 ```sh
-$ caesarlib encipher "Carolus Magnus" --offset 87
-OmDAxGE YmszGE
+$ cargo run -- --method encipher --offset 87 --text "Carolus Magnus"
+Result: lJaXUdb vJPWdb
+With Offset: 87
 
-$ caesarlib decipher "OmDAxGE YmszGE" --offset 87
-Carolus Magnus
+$ cargo run -- --method decipher --offset 87 —text "lJaXUdb vJPWdb"
+Result: Carolus Magnus
+With Offset: 87
 
-$ caesarlib encipher "Carolus Magnus" --offset 87 | caesarlib decipher --offset 87
```

Comment:
./target/debug/caesarlib --method encipher --offset 87 --text "toot" | grep Offset: | cut -d " " -f2 | xargs ./target/debug/caesarlib --method decipher --offset 87 --text | grep Offset: | cut -d " " -f2

It looks more ugly actually...